curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md

```diff
@@ -18,15 +18,15 @@ localeTitle: Use seletores CSS para elementos de estilo
 ```yml
 tests:
   - text: Remova o atributo style do seu elemento <code>h2</code> .
-    testString: 'assert(!$("h2").attr("style"), "Remove the style attribute from your <code>h2</code> element.");'
+    testString: assert(!$("h2").attr("style"));
   - text: Crie um elemento de <code>style</code> .
-    testString: 'assert($("style") && $("style").length > 1, "Create a <code>style</code> element.");'
+    testString: assert($("style") && $("style").length >= 1);
```

Comment:
I suggest the text translated
```suggestion
    testString: 'assert($("style") && $("style").length > 1, "Crie um elemento <code>style</code>.");'
```

Reply:
We don't need to add assert messages.  Instead, the `text` section is the assert message on the preceding line should be adjusted as needed.